internal/nexodus/nexodus.go

```diff
@@ -518,6 +532,12 @@ func (nx *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		}
 	}
 
+	if nx.exitNode.exitNodeOriginEnabled {
+		if err := nx.exitNodeOriginSetup(); err != nil {
+			return fmt.Errorf("failed to setup this device as an exit-node: %w", err)
+		}
+	}
```

Comment:
This will need to be integrated into the reconcile loop instead of only at startup to ensure that the proper configuration stays in place.

Reply:
Good call. This is big enough I'd like to do in a followup. The code changes are relatively minimal, but I do want to allocate additional time testing the scenario (particularly exit node client side) of it flipping between 5g/wired.

https://github.com/nexodus-io/nexodus/issues/1408